### ISSUE TITLE
Error: Cannot find module '@actions/core'

### ISSUE THREAD

**Issue by mattiasgustavsson-sj** [Azure/manage-azure-policy#58]
This is an issue when we try to publish our latest policy updates.

Our best guess is that it is related to last commit by @tauhid621. See more info in below image:

![image](https://user-images.githubusercontent.com/78361328/114873048-3535c300-9dfb-11eb-93b0-577873e4b36d.png)

**Comment by asos-alanscott** [comment]
Can confirm, same issue here since this PR #56 

---

Previous commit works as expected
`uses: azure/manage-azure-policy@c693881e3b39a41eadc4b74de0ab032a73eeab3b`


---

Latest commit 
`uses: azure/manage-azure-policy@v0`
      
Fails with 
Error: Cannot find module '@actions/core'

**Comment by tauhid621** [comment]
Updated the v0 tag.
No idea how my commit broke things as the tag was not updated.

**Comment by sbugalski** [comment]
@tauhid621 
I receive `Error while deploying policies` on v0, works good on @c693881e3b39a41eadc4b74de0ab032a73eeab3b. Don't know if it is related.

```
Run azure/manage-azure-policy@v0
  with:
    paths: config-files/policy/policies/**
  config-files/policy/initiatives/**
  
  env:
    User: ***
    Password: ***
    TenantId: ***
----------------------------------------------------------------------------------------------------
Error : Error occurred while fetching policies from Azure.
----------------------------------------------------------------------------------------------------
Error: Error while deploying policies.
```

**Comment by tauhid621** [comment]
@sbugalski This can be related to https://github.com/Azure/manage-azure-policy/pull/56

Is there some error while fetching policies?

**Comment by sbugalski** [comment]
@tauhid621 - doesn't look like. Please note latest official version still doesn't work for over 3 weeks 👎🏻 

Compare runs azure/manage-azure-policy@v0 vs azure/manage-azure-policy@c693881e3b39a41eadc4b74de0ab032a73eeab3b

```
##[debug]Evaluating condition for step: 'Create or Update Azure Policies'
##[debug]Evaluating: (success() && (github.ref == 'refs/heads/main'))
##[debug]Evaluating And:
##[debug]..Evaluating success:
##[debug]..=> true
##[debug]..Evaluating Equal:
##[debug]....Evaluating Index:
##[debug]......Evaluating github:
##[debug]......=> Object
##[debug]......Evaluating String:
##[debug]......=> 'ref'
##[debug]....=> 'refs/heads/main'
##[debug]....Evaluating String:
##[debug]....=> 'refs/heads/main'
##[debug]..=> true
##[debug]=> true
##[debug]Expanded: (true && ('refs/heads/main' == 'refs/heads/main'))
##[debug]Result: true
##[debug]Starting: Create or Update Azure Policies
##[debug]Loading inputs
##[debug]Loading env
Run azure/manage-azure-policy@v0
[...]
##[debug]Comparing Hash for policy id : /providers/Microsoft.Management/managementGroups/<redacted>/providers/Microsoft.Authorization/policyAssignments/<redacted> : true
##[debug]----------------------------------------------------------------------------------------------------
----------------------------------------------------------------------------------------------------
Error : Error occurred while fetching policies from Azure.
----------------------------------------------------------------------------------------------------
Error: Error while deploying policies.
##[debug]Node Action run completed with exit code 1
##[debug]AZURE_HTTP_USER_AGENT='GITHUBACTIONS_ManageAzurePolicy_a7dbc8a83a321e1a68ff9c33c16ed43f9a169acb78c920cd06937f7f2189f86e'
##[debug]Finishing: Create or Update Azure Policies
```

```
##[debug]Evaluating condition for step: 'Create or Update Azure Policies'
##[debug]Evaluating: (success() && (github.ref == 'refs/heads/main'))
##[debug]Evaluating And:
##[debug]..Evaluating success:
##[debug]..=> true
##[debug]..Evaluating Equal:
##[debug]....Evaluating Index:
##[debug]......Evaluating github:
##[debug]......=> Object
##[debug]......Evaluating String:
##[debug]......=> 'ref'
##[debug]....=> 'refs/heads/main'
##[debug]....Evaluating String:
##[debug]....=> 'refs/heads/main'
##[debug]..=> true
##[debug]=> true
##[debug]Expanded: (true && ('refs/heads/main' == 'refs/heads/main'))
##[debug]Result: true
##[debug]Starting: Create or Update Azure Policies
##[debug]Loading inputs
##[debug]Loading env
Run azure/manage-azure-policy@c693881e3b39a41eadc4b74de0ab032a73eeab3b
[...]
##[debug]Comparing Hash for policy id : /providers/Microsoft.Management/managementGroups/<redacted>/providers/Microsoft.Authorization/policyAssignments/<redacted>: true
##[debug]----------------------------------------------------------------------------------------------------
##[debug]"/usr/bin/az" account get-access-token --resource=https://management.azure.com
::add-mask::***
##[debug][POST]https://management.azure.com/batch?api-version=2019-09-01
##[debug]----------------------------------------------------------------------------------------------------
##[debug]Status of batch calls:
##[debug]----------------------------------------------------------------------------------------------------
##[debug]Name : 0. Status : 400
##[debug]----------------------------------------------------------------------------------------------------
##[debug]End
##[debug]----------------------------------------------------------------------------------------------------
##[debug]"/usr/bin/az" account get-access-token --resource=https://management.azure.com
::add-mask::***
##[debug]----------------------------------------------------------------------------------------------------
##[debug]No role assignments needs to be created
##[debug]----------------------------------------------------------------------------------------------------
```

**Comment by tauhid621** [comment]
Fixed in #63
